pkgs/desktops/gnome/extensions/nautilus-open-any-terminal/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, pkg-config
+, python3Packages
+, dbus
+, dconf
+, gobject-introspection
+, substituteAll
+, fetchFromGitHub
+, gnome3
+, glib
+, gtk3
+, python3
+, gsettings-desktop-schemas
+, wrapGAppsHook
+}:
+
+# nix-env-pwd-install gnomeExtensions.nautilus-open-any-terminal
+
+python3Packages.buildPythonPackage rec {
+  pname = "nautilus-open-any-terminal";
+  version = "0.2.15";
+
+  src = fetchFromGitHub {
+    owner = "Stunkymonkey";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-cc6Lh5XeAuU5Os4eJ0QcL6XJYB6DqxeUGaOf6m1OnpY=";
+  };
+
+  nativeBuildInputs = [
+    glib
+    gobject-introspection
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    dbus
+    dconf
+    gnome3.nautilus
+    gnome3.nautilus-python
+    gsettings-desktop-schemas
+    gtk3
+    python3
+    python3.pkgs.pygobject3
+    gobject-introspection
+  ];
+
+  patches = [ ./hardcode-gsettings.patch ];
```

Comment:
We prefer having `patches` after `src`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887 or https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md.

Reply:
nice to know. thanks